### PR TITLE
Reusable bench-db & bench-keyring

### PR DESCRIPTION
**Pull request by NikVolf**
This puts benchmarking db tools in `node-testing/bench` module for reuse

### CODE REVIEW

File: bin/node/testing/src/bench.rs [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+					needs_justification: false,
+					bad_justification: false,
+					needs_finality_proof: false,
+					is_new_best: true,
+				}
+			)
+		);
+
+		assert_eq!(self.client.chain_info().best_number, 1);
+    }
```
Comment (bkchr) [inline bin/node/testing/src/bench.rs]: Whitespaces.

File: bin/node/testing/src/bench.rs [hunk 2]
```diff
@@ -0,0 +1,424 @@
...
+
+    /// Database path for the current context.
```
Comment (bkchr) [inline bin/node/testing/src/bench.rs]: More.

File: bin/node/testing/src/bench.rs [hunk 3]
```diff
@@ -0,0 +1,424 @@
...
+            Some(node_runtime::WASM_BINARY),
+            self.collect_account_ids(),
+        )
+    }
+}
+
+/// Profile for exetion strategies.
+#[derive(Clone, Copy, Debug)]
+pub enum Profile {
+    /// As native as possible.
```
Comment (bkchr) [inline bin/node/testing/src/bench.rs]: Whitespaces.

File: bin/node/testing/src/bench.rs [hunk 4]
```diff
@@ -0,0 +1,424 @@
...
+				}
+			}
+			None => UncheckedExtrinsic {
+				signature: None,
+				function: xt.function,
+			},
+		}
+    }
+
+    /// Generate genesis with accounts from this keyring endowed with some balance.
```
Comment (bkchr) [inline bin/node/testing/src/bench.rs]: More.

File: bin/node/testing/src/bench.rs [hunk 5]
```diff
@@ -0,0 +1,424 @@
...
+		let (client, backend) = Self::bench_client(directory_guard.path(), profile, &keyring);
+
+		BenchContext {
+			client, backend, db_guard: directory_guard,
+		}
+	}
+}
+
+impl BenchKeyring {
+    /// New keyring.
```
Comment (bkchr) [inline bin/node/testing/src/bench.rs]: Whitespaces

**Comment by NikVolf** [comment]
> A lot of whitespaces that need to be removed.

sorry, vs code.. fixed!

**Pull request merged**